penguin/integrations/mcp/server_tools/pm.py

```
@@ -0,0 +1,280 @@
+            budget_tokens=_optional_int(arguments.get("budget_tokens")),
+            budget_minutes=_optional_int(arguments.get("budget_minutes")),
+            workspace_path=_optional_path(arguments.get("workspace_path")),
+            **metadata,
+        )
+        changed = False
+        for attr in ("priority", "start_date", "due_date"):
+            if attr in arguments and arguments[attr] is not None:
+                setattr(project, attr, arguments[attr])
+                changed = True
+        if changed:
+            project.updated_at = datetime.utcnow().isoformat()
```

Ruff (0.15.12)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)